packages/in_app_purchase/example/lib/main.dart

```diff
@@ -120,26 +165,51 @@ class _MyAppState extends State<MyApp> {
     // This loading previous purchases code is just a demo. Please do not use this as it is.
     // In your app you should always verify the purchase data using the `verificationData` inside the [PurchaseDetails] object before trusting it.
     // We recommend that you use your own server to verity the purchase data.
-    Map<String, PurchaseDetails> purchases = Map.fromEntries(((await connection
-                .queryPastPurchases())
-            .pastPurchases)
-        .map((PurchaseDetails purchase) =>
-            MapEntry<String, PurchaseDetails>(purchase.productId, purchase)));
-
+    Map<String, PurchaseDetails> purchases = Map.fromEntries(
+        ((await connection.queryPastPurchases()).pastPurchases)
+            .map((PurchaseDetails purchase) {
+      print('restored productID: ${purchase.productID}');
```

Comment:
nit: I do think we'd be better off without all these logs, especially the ones just saying that a particular method has fired.